gauge_energy.hh

```diff
@@ -9,17 +9,20 @@
 //
 // checked for gauge invariance
 
-template<class T> double gauge_energy(gaugeconfig<T> &U) {
+template<class T> double gauge_energy(gaugeconfig<T> &U, bool spacial=false) {
 
   double res = 0.;
 #ifdef _USE_OMP_
   int threads = omp_get_max_threads();
-  static double * omp_acc = new double[threads];
+  double * omp_acc = new double[threads];
 #pragma omp parallel
   {
     int thread_num = omp_get_thread_num();
 #endif
     double tmp = 0.;
+    
+size_t startmu=0;
+if(spacial){startmu=1;};
  
 #pragma omp for
```

Comment:
I will look into if this conflicts with the case where openMP is not used, but I agree that it would be nice to use something simpler at least in the trivially parallelizable cases. So the same should also be true e.g. for the Wilson-Loops. I am not sure if the same is possible for the sweep-function, I will also look into this.